app/helpers/results_helper.rb

```diff
@@ -48,7 +48,7 @@ def pdf_friendly_h3(text, is_pdf)
     end
   end
 
-  def pdf_friendly_p_element(text, is_pdf, additional_class)
+  def pdf_friendly_p_element(text, is_pdf, additional_class = nil)
```

Comment:
question - this makes sense. Would we want to get rid of the `nil` in all the place we are calling `pdf_friendly_p_element` or shoudl we leave them there?

Reply:
Ah, yep I hadn't spotted that we were calling this with a bunch of explicit `nil`s. That's not ideal, so I've added a tidy-up commit to stop doing that.